test/npmTests.ts

```diff
@@ -43,4 +45,43 @@ describe("npm.ts", function () {
       assert.strictEqual(npmCommand, "npm.cmd");
     });
   });
+
+  describe("npm()", function () {
+    it("with unrecognized command", function () {
+      const result: RunResult = npm("foo");
+      assert(result);
+      assert.strictEqual(result.exitCode, 1);
+      assert.deepEqual(getLines(result.stdout), [
+        "",
+        "Usage: npm <command>",
+        "",
+        "where <command> is one of:",
+        "    access, adduser, audit, bin, bugs, c, cache, ci, cit,",
+        "    completion, config, create, ddp, dedupe, deprecate,",
+        "    dist-tag, docs, doctor, edit, explore, get, help,",
+        "    help-search, hook, i, init, install, install-test, it, link,",
+        "    list, ln, login, logout, ls, outdated, owner, pack, ping,",
+        "    prefix, profile, prune, publish, rb, rebuild, repo, restart,",
+        "    root, run, run-script, s, se, search, set, shrinkwrap, star,",
+        "    stars, start, stop, t, team, test, token, tst, un,",
+        "    uninstall, unpublish, unstar, up, update, v, version, view,",
+        "    whoami",
+        "",
+        "npm <command> -h  quick help on <command>",
+        "npm -l            display full usage info",
+        "npm help <term>   search for help on <term>",
+        "npm help npm      involved overview",
+        "",
+        "Specify configs in the ini-formatted file:",
+        "    C:\\Users\\daschult\\.npmrc",
```

Comment:
This is not going to pass on my machine ;)

Reply:
Agreed. I'll fix this.